pom.xml

```diff
@@ -6,7 +6,7 @@ Based on the cool scp plugin.</description>
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.424</version>
+		<version>1.596</version>
```

Comment:
Not at all, I figured it couldn't hurt to update this version as well. But maybe I was wrong.
Does it introduce incompatibilities with earlier versions of Jenkins?
Sorry, I'm not familiar with Jenkins plugin development.